hack/lib/util.sh

```diff
@@ -244,18 +244,18 @@ kube::util::remove-gen-docs() {
 kube::util::group-version-to-pkg-path() {
   staging_apis=(
   $(
-    pushd ${KUBE_ROOT}/staging/src/k8s.io/api > /dev/null
-      find . -name types.go | xargs -n1 dirname | sed "s|\./||g" | sort
-    popd > /dev/null
-  )
-  )
+    cd ${KUBE_ROOT}/staging/src/k8s.io/api &&
+    find . -name types.go -exec dirname {} \; | sed "s|\./||g" | sort
+  ))
 
   local group_version="$1"
 
-  if [[ " ${staging_apis[@]} " =~ " ${group_version/.*k8s.io/} " ]]; then
-    echo "vendor/k8s.io/api/${group_version/.*k8s.io/}"
-    return
-  fi
+  for api in "${staging_apis[@]:-}"; do
```

Comment:
this changes the semantics of the function -- is it correct?

Reply:
The semantics changes, yes. The original issue in the code is not that severe -- it requires specifically-crafted code to get a false match from the regexp, even though the array is concatenated to a single string. This is bit complex function and I'm not fully certain the first check is supposed to only handle the `apiname/version` case. Let's drop this patch just to be on the safe side.